### PR TITLE
[New Feature] Adding the items endpoint support

### PR DESCRIPTION
**Pull request by Feromond**
Added items and created classes to handle the initial response and the specific sky block item that is within the large list of items. There could still be some extra methods added to allow for some more specific filters very similar list_item_categories  but for some of the other things if we want.

### CODE REVIEW

File: hypixel_api_lib/Items.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+class Items:
+    """
+    Handles fetching and managing all the items from the API.
+    
+    Attributes:
+        api_endpoint (str): The endpoint URL to fetch the items data.
+        items (dict of str: SkyBlockItem): A dictionary of item IDs to SkyBlockItem objects.
+    """
+    
+    def __init__(self, api_endpoint=ITEMS_API_URL):
```
Comment (desobob11) [inline hypixel_api_lib/Items.py:63]: nice touch

**Review by desobob11 — COMMENTED** [review]
Super nice! I like the documentation

File: hypixel_api_lib/Items.py (hunk 1 above)
Comment (Feromond) [reply hypixel_api_lib/Items.py:63]: @desobob11 You think its good to merge now then? Approve if you are happy with it and then we can complete this PR

**Comment by desobob11** [comment]
Approve AF

**Review by desobob11 — APPROVED** [review]
Approved AF

**Pull request merged**